pkg/osquery/tables/apple_silicon_security_policy/parser.go

```diff
@@ -0,0 +1,171 @@
+package apple_silicon_security_policy
+
+import (
+	"bufio"
+	"io"
+	"regexp"
+	"strings"
+)
+
+// This regexp gets matches for an arbitrary property name, and a four-character code (4CC) sequence
+var fourCharCodeRegexp = regexp.MustCompile(`^((.*) (?:\((.{4})\)))`)
+
+// This regexp gets matches for the volume group UUID following the text "Local policy for volume group"
+var volumeGroupRegexp = regexp.MustCompile("^((?:(?:.*) for volume group))(.*):")
```

Comment:
Are there extra non-capture groups here?

vs: `"for volume group(.*):"`

Reply:
For some reason I got really non-capture-group happy when I first wrote this. Simplified both regexps now.